test/models/test_portal_configuration.rb

```diff
@@ -1,27 +0,0 @@
-require_relative '../test_case'
-
-class TestPortalConfiguration < LinkedData::TestCase
-
-  def test_read_portal_config
```

Comment:
this test need to stay, just replace `LinkedData::Models::PortalConfig.current_portal_config` but the model that you created.

Reply:
i'll add it to the test of the file `test_artefact_catalog.rb` that covers the test related to the SemanticArtefactCatalog